### DISCUSSION TITLE
Azure Synapse Dedicated SQL Pool and DBeaver

### DISCUSSION THREAD

**Discussion by lostsole** [dbeaver/dbeaver#20230] [General]
Microsoft has has told me that DBeaver is running a command that is not supported, Can anyone tell me how to stop DBeaver from running this command.  In the last 30 days 20,000 querys of this has been run.

the resource you are using is Azure Synapse Dedicated SQL Pool (Formerly SQL DW). This SQL Pool version does not support sys.dm_db_resource_stats Dynamic Management View. This DMV is supported in Azure SQL Database and Azure SQL Managed Instance only.

DBeaver Metadata query:
select
AVG(avg_cpu_percent),
MAX(avg_cpu_percent),
AVG(avg_data_io_percent),
MAX(avg_data_io_percent),
AVG(avg_log_write_percent), 
MAX(avg_log_write_percent),
AVG(avg_memory_usage_percent),
MAX(avg_memory_usage_percent),
Avg(dtu_limit) 
from (
SELECT * FROM sys.dm_db_resource_stats where end_time > getdate()
) AS T
 
WHERE sys.dm_db_resource_stats is not supported by Azure Synapse Dedicated SQL Pool

**Comment by LonwoLonwo** [comment]
Hello @lostsole 

I do not see this query in DBeaver's code.

You can turn it off if it is connected with the statistics reading.

![2023-06-07 20_04_52-DBeaver Ultimate 23 1 0 - _SAlesforce 2_ Script-219  Script-219](https://github.com/dbeaver/dbeaver/assets/45152336/3fbf70c2-6690-4d53-a16c-44a23712ef07)

**Comment by lostsole** [comment]
I will work with the our developers tomorrow that uses DBeaver to see if the sessions is showing the query. and see if this will turn this off.  I will let you know what I find and if this setting change blocks the system DMV from running
